libpod/runtime.go

```diff
@@ -227,9 +227,8 @@ func (r *Runtime) Shutdown(force bool) error {
 
 	_, err := r.store.Shutdown(force)
 	if err != nil {
-		return err
+		logrus.Errorf("Error shutting down storage: %v", err)
```

Comment:
Maybe save this as lastError similar to the code in cmd/kpod, so we don't exit cleanly if r.state.Close() succeeds but r.store.Shutdown() fails